Improve the README

chasenlehara opened the pull request:
- Remove `> ` from in front of commands so they’re easier to copy & paste
- Improve some of the instructions
- Add a screenshot of the “finished” app

chasenlehara committed:
Improve the README

- Remove `> ` from in front of commands so they’re easier to copy & paste
- Improve some of the instructions
- Add a screenshot of the “finished” app

justinbmeyer commented:
So I think we need something to signal "this is code" vs "this is something you put in your terminal". A LONG time ago, people would actually make this mistake, hence why we put `>` in most of our guides.


Perhaps we can do something like: ```cmd ... which should add `cmd` to the class name and give these things a different style.

chasenlehara committed:
Add a language to the code examples in the README

Change based on this feedback: https://github.com/stealjs/quick-start/pull/9#issuecomment-274384040

chasenlehara commented:
Today I learned about the full list of languages that GitHub supports: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

I marked the command line examples as `shell`; it only makes a little bit of a difference:
<img width="137" alt="screen shot 2017-01-23 at 10 02 34 am" src="https://cloud.githubusercontent.com/assets/10070176/22216250/2aba7d4e-e153-11e6-9843-c5b2a87dc8f1.png">
<img width="138" alt="screen shot 2017-01-23 at 10 02 44 am" src="https://cloud.githubusercontent.com/assets/10070176/22216252/2d1c0d6e-e153-11e6-9c6c-689109c63fa8.png">
